src/app/App.js

```diff
@@ -29,6 +29,7 @@ export default class AppProvider extends React.Component<PropsType> {
 		if ( initialTitle === undefined ) {
 			initialTitle = 'Welcome to Gutenberg!';
 		}
+
```

Comment:
Nit: let's remove this empty line diff. Also the same in the `App.test.js` file.

Reply:
👍

Reply:
Seems addressed by 7bd3b0b76a5d17a6d023467b07fb687d0f605191.